wgsl/index.bs

```diff
@@ -1082,7 +1073,7 @@ ordering requirements for a loop in SPIR-V.
     const a : i32 = 2;
     var i : i32 = 0; <1>
     loop {
-      break if (i >= 4);
```

Comment:
I'd lean towards the #705 solution here and we drop the `{}`s in the limited cases.